wallet/src/db/encrypted/mod.rs

```diff
@@ -72,6 +72,15 @@ impl Database for EncryptedDb {
         }
     }
 
+    fn delete<K>(&self, key: &K) -> Result<()>
+    where
+        K: AsRef<[u8]>,
+    {
+        self.as_ref().delete(key)?;
+
+        Ok(())
+    }
+
```

Comment:
This key is not correct! :) Maybe we should take this delete feature from this PR as it is not used.